rclpy/package.xml

```diff
@@ -14,9 +14,13 @@
   <depend>rmw_implementation</depend>
   <depend>rcl</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
+
   <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_index_python</test_depend>
```

Comment:
If the package doesn't need `ament_index_python` a build time of the tests (which it shouldn't?) it doen't need a *test* dependency since it is already an *exec* dependency